course/templates/course/course_base.html

```diff
@@ -13,12 +13,20 @@
 {% endfor %}
 {% endblock %}
 
+{% block container_opening_tag %}
+<div class="site-content container-fluid" id="content">                
+{% endblock %}
+
+{% block container_closing_tag %}
+</div>                
+{% endblock %}
+
```

Comment:
Why are you using one block for the opening tag and another for the closing tag? is not it better to use only one block?

Reply:
I couldn't see how to make them overridable in Django's templating language while preserving whatever children they have. It's probably possible, but this was the only way I managed to get it to work after a couple of hours of Googling.

Can you provide a minimal example of your approach?

Reply:
I do not have an example, I was only curious about what you wanted to accomplish with this opening and closing blocks. Now, it is more clear, and I think it is fine if you do it in this way.